airflow/models/connection.py

```diff
@@ -473,8 +473,18 @@ def get_connection_from_secrets(cls, conn_id: str) -> Connection:
                 if hasattr(sys.modules.get("airflow.sdk.execution_time.task_runner"), "SUPERVISOR_COMMS"):
                     # TODO: AIP 72: Add deprecation here once we move this module to task sdk.
                     from airflow.sdk import Connection as TaskSDKConnection
-
-                    return TaskSDKConnection.get(conn_id=conn_id)
+                    from airflow.sdk.exceptions import AirflowRuntimeError, ErrorType
```

Comment:
Thanks for making this change. I added some tests in the original PR #47593, could you add a case there too?

Reply:
Added!